Jusibe/JusibeClient.cs

```diff
@@ -85,5 +85,56 @@ public class JusibeClient
                 }
             });
         }
+        
+        public async Task<ResponseModel> SendSms(RequestModel model) {
+            var request = (HttpWebRequest)WebRequest.Create(this.config.ResolveURL("send_sms"));
+            request.Method = Constants.POST;
+            request.ContentType = Constants.X_WWW_FORM_URL_ENCODED;
+            request.Credentials = this.config.Credentials;
+            request.Headers.Add("Authorization", this.config.AuthorizationHeader);
+            
+            return await Task.Run(() => {

```

Comment:
Try changing this line to `return await Task.Run(async () => {` 

That should fix the build error.